content/posts/2021/reana-client-python-api.md

```diff
@@ -0,0 +1,170 @@
+---
+title: "How to run workflows using reana-client Python API"
+date: 2021-02-22T15:00:00+01:00
+tags:
+  - API
+  - client
+  - Python
+---
+
+Here you can find a step by step guide on how to submit workflows to
+REANA using the `reana-client` Python API. This is a simple example,
+for illustrative purposes, using the serial workflow engine and a
+simple [ROOT6 / RooFit analysis](https://github.com/reanahub/reana-demo-root6-roofit/) example.
+We will create a new workflow, run it and retrieve its results using
+the `reana-client` Python API.
+<!--more-->
+
+Install `reana-client`:
+```
+$ pip install reana-client==0.7.0
```

Comment:
Should we point to [0.7.2](https://github.com/reanahub/reana-client/releases/tag/0.7.2)? it seems a bit weird to point to an old version.